root/account/applications/Index.js

```diff
@@ -122,13 +122,13 @@ const Index = ({
     <p>
       {exp.l(
         `Do you want to develop an application that uses the
-         {ws|MusicBrainz web service}? {register|Register an application}
+         {api|MusicBrainz API}? {register|Register an application}
          to generate OAuth tokens. See our {oauth2|OAuth documentation}
          for more details.`,
         {
+          api: '/doc/MusicBrainz_API',
           oauth2: '/doc/Development/OAuth2',
           register: '/account/applications/register',
-          ws: '/doc/Development/XML_Web_Service/Version_2',
         },
       )}
```

Comment:
If you are renaming a key, I would suggest to rename all keys so as to make them more transparent to translators: `mb_api_doc_url`, `oauth2_doc_url`, `register_url`.